utils/rpms/daos.spec

```diff
@@ -121,6 +121,14 @@ BuildRequires: libuct-devel
 BuildRequires: ucx-devel
 %endif
 
+# Needed for debugging tasks
+%if (0%{?rhel} >= 8)
+BuildRequires: libasan
+%endif
+%if (0%{?suse_version} > 0)
+BuildRequires: libasan8
+%endif
+%
```

Comment:
This is a stray `%` that will cause parsing problems.

Reply:
- [ ] Fix spec file syntax

Reply:
> * [x]  Fix spec file syntax

Fixed with commit ce48861afb63168da31c2fc59f094719b008e9f7